### DISCUSSION TITLE
How to increase the max search count?

### DISCUSSION THREAD

**Discussion by kcaashish** [folke/noice.nvim#618] [Q&A]
I am using LazyVim and have this added to Lualine. I just cannot figure out how to increase the max count from 99.
```lua
table.insert(opts.sections.lualine_x, 2, {
  function()
    -- trim the message to include only the count in between large brackets
    local count = require("noice").api.status.search.get()
    return "Count: " .. string.match(count, "%[(.*)%]")
  end,
  cond = function()
    return package.loaded["noice"] and require("noice").api.status.search.has()
  end,
  color = require("lazyvim.util").ui.fg("special"),
})
```

Saw that we could do it using `"searchcount"` directly with lualine and just adding `maxcount=999`, but in that case I don't know how to manipulate the output to fit my need.

**Comment by folke** [comment]
noice just uses whatever Neovim gives us through `ui_attach`
